npm/private/pnpm.bzl

```diff
@@ -499,6 +534,11 @@ def _convert_v9_packages(packages, snapshots):
             resolution = package_data.get("resolution"),
         )
 
+        # Match v6 lockfile behaviour with local dependency keys
+        if "file:" in package_key:
```

Comment:
This as well. I'd like to see this change along with a test demonstrating the side-effects...